lib/MusicBrainz/Server/Controller/WS/2/Recording.pm

```diff
@@ -242,12 +246,22 @@ sub recording_submit : Private
             on_full => sub {
                 my $contents = shift;
                 try {
-                    $c->model('Edit')->create(
+                    my $rem = $c->model('Edit')->create(
```

Comment:
(curious, what does `$rem` stand for? 🤔)

Reply:
:D I think it was meant as "removal" where I copied it from, lol. Will amend to $edit or whatever.